src/Zaaksysteem/Entities/Casts/Related/Rollen.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OWC\Zaaksysteem\Entities\Casts\Related;
+
+use OWC\Zaaksysteem\Entities\Entity;
+use OWC\Zaaksysteem\Support\Collection;
+use OWC\Zaaksysteem\Endpoint\Filter\RollenFilter;
+
+use function OWC\Zaaksysteem\Foundation\Helpers\resolve;
+
+class Rollen extends ResourceCollection
+{
+    public function resolveRelatedResourceCollection(Entity $entity): Collection
+    {
+        $statussenEndpoint = resolve($this->clientName)->rollen();
```

Comment:
Is dit altijd een collection?

Reply:
Ja, want de `filter()` method gebruikt de `getPagedCollection()` method in `Endpoint. En die returned altijd een `PagedCollection` wat een uitbreiding is op de `Collection` class. Als er geen resultaten zijn dan is de collection leeg.

Ik zie wel dat de variabele naam even geupdatet moet worden.